### DISCUSSION TITLE
What is ASMS in AsmsTracks?

### DISCUSSION THREAD

**Discussion by rupinderjeet** [jhomlala/betterplayer#1043] [Q&A] [answered]
I do not know what ASMS is. I want to find more about this. Can you let me know full form or any reference for this? Or is it just a random name used to group code classes?

Searching for "hls asms track" only brings up this library. I have searched for this in discussions as well.

I do not expect any help on code from you since I know that you couldn't continue the development. I just need some information to do it myself in my free time. Hopefully you'll reply with some help.

**Comment by noboru-i** [comment, marked as answer]
I found a document.
https://pub.dev/documentation/better_player/latest/better_player/BetterPlayerDataSource/useAsmsSubtitles.html

> ASMS - Adaptive Streaming Media Sources
